src/components/date-facet.tsx

```diff
@@ -0,0 +1,58 @@
+import {DateFacet as HeadlessDateFacet, DateFacetValue} from '@coveo/headless';
+import {FunctionComponent, useEffect, useState} from 'react';
+import dayjs from 'dayjs';
+
+interface DateFacetProps {
+  controller: HeadlessDateFacet;
+  title: string;
+}
+
+export const DateFacet: FunctionComponent<DateFacetProps> = (props) => {
+  const {controller} = props;
+  const [state, setState] = useState(controller.state);
+
+  useEffect(() => controller.subscribe(() => setState(controller.state)), []);
+
+  const parseDate = (date: string) => {
+    return dayjs(date, 'YYYY/MM/DD@HH:mm:ss');
+  };
+
+  const format = (dateStr: string) => {
+    return parseDate(dateStr).format('MMMM D YYYY');
+  };
```

Comment:
This could easily be combined into one function, since only `format` is actually called in the render function